docs/internals/generic/fake.md

```diff
@@ -0,0 +1,39 @@
+---
+title: "Configure a Fake Generic Service"
+linkTitle: "fake"
+weight: 10
+type: "docs"
+description: "Configure a fake generic service."
+tags: ["generic", "services"]
+---
+
+Configure a `fake` generic service to test implementing a generic service on your machine:
+
+{{< tabs >}}
+{{% tab name="Config Builder" %}}
+
+Navigate to the **Config** tab of your machine's page in [the Viam app](https://app.viam.com).
+Click on the **Services** subtab and click **Create service**.
+Select the `generic` type, then select the `fake` model.
+Enter a name for your generic service and click **Create**.
+
+![An example configuration for a fake generic service in the Viam app Config Builder.](/components/generic/fake-generic-ui-config.png)
+
+{{% /tab %}}
+{{% tab name="JSON Template" %}}
+
+```json {class="line-numbers linkable-line-numbers"}
+{
+  "name": "<your-fake-generic-component-name>",
+  "model": "fake",
+  "type": "generic",
+  "namespace": "rdk",
+  "attributes": {}
+}
+```
+
+{{% /tab %}}
+{{< /tabs >}}
+
+No attributes are available for fake generic service.
+See [GitHub](https://github.com/viamrobotics/rdk/blob/main/services/generic/fake/generic.go) for API call return specifications.
```

Comment:
Opted not to include content about "controlling a generic service", as is present for generic components, as services are not generally controllable in likewise fashion.

Reply:
That seems good to me.